src/vnsw/agent/services/dhcpv6_handler.h

```diff
@@ -164,21 +164,13 @@ struct Dhcpv6Options {
 
     uint16_t code;
     uint16_t len;
-#ifndef _WINDOWS
     uint8_t  data[0];
-#else
-	uint8_t  data[1];
-#endif
 };
 
 struct Dhcpv6Hdr {
     uint8_t     type;
     uint8_t     xid[3];
-#ifndef _WINDOWS
-    Dhcpv6Options options[0];
-#else
-	Dhcpv6Options options[];
-#endif
+    // followed by options
```

Comment:
We probably need a better verbose comment here, explaining how this is padded now, so that people (especially Juniper reviewers) don't suddenly thing `options[]` missing is a mistake.

Reply:
This is consistent with similar comments in the same file (lines 182 and 206).